experimental/ner_spancat_compare/project.yml

```diff
@@ -0,0 +1,315 @@
+  make it strict and we miss might out on some true positives.
+
+  |           |   Precision |   Recall |   F-score |
+  |-----------|-------------|----------|-----------|
+  | DNA       |    0.695853 | 0.360382 |  0.474843 |
+  | protein   |    0.772816 | 0.519243 |  0.621147 |
+  | cell_line |    0.773256 | 0.303653 |  0.436066 |
+  | cell_type |    0.759674 | 0.61755  |  0.681279 |
+  | RNA       |    0.771429 | 0.247706 |  0.375    |
+  | **Overall**| 0.7562334996 | 0.4709535988 | 0.5804345379 |
+
+  #### NER Results
+
+  NER performs well against SpanCat for all entity types, but note that this
+  process entails training five (5) models per entity type. This might work if
+  you have a small number of entities, but can be computationally heavy if you
+  have a lot.
+
+  |           |   Precision |   Recall |   F-score |
+  |-----------|-------------|----------|-----------|
+  | DNA       |    0.735721 | 0.630705 |  0.679178 |
+  | protein   |    0.75833  | 0.718349 |  0.737798 |
+  | cell_line |    0.738462 | 0.567376 |  0.641711 |
+  | cell_type |    0.777778 | 0.721137 |  0.748387 |
+  | RNA       |    0.855422 | 0.651376 |  0.739583 |
+
+  Since we have five (5) separate models in NER, what we can do afterwards is
+  combine them into a single `Doc` that transfers `doc.ents` to `doc.spans`. Since
+  the tokens are the same, we don't need to worry about misalignments and the like.
+
+vars:
+  name: "ner_spancat_compare"
+  version: "1.0.0"
+  gpu_id: -1
+  # Labels
+  dna: "DNA"
+  rna: "RNA"
+  cell_line: "cell_line"
+  cell_type: "cell_type"
+  protein: "protein"
+
+directories:
+  - "assets"
+  - "configs"
+  - "corpus"
+  - "corpus/ner"
+  - "metrics"
+  - "scripts"
+  - "training"
+
+workflows:
+  all:
+    - "install"
+    - "convert"
+    - "create-ner"
+    - "train-ner"
+    - "train-spancat"
+    - "evaluate-ner"
+    - "evaluate-spancat"
+  spancat:
+    - "install"
+    - "convert"
+    - "train-spancat"
+    - "evaluate-spancat"
+  ner:
+    - "install"
+    - "convert"
+    - "create-ner"
+    - "train-ner"
+    - "evaluate-ner"
+
+assets:
+  - dest: "assets/train.iob2"
+    description: "The training dataset for GENIA in IOB format."
+    url: https://github.com/thecharm/boundary-aware-nested-ner/blob/master/Our_boundary-aware_model/data/genia/genia.train.iob2
+  - dest: "assets/dev.iob2"
+    description: "The evaluation dataset for GENIA in IOB format."
+    url: https://github.com/thecharm/boundary-aware-nested-ner/blob/master/Our_boundary-aware_model/data/genia/genia.dev.iob2
+  - dest: "assets/test.iob2"
+    description: "The test dataset for GENIA in IOB format."
+    url: https://github.com/thecharm/boundary-aware-nested-ner/blob/master/Our_boundary-aware_model/data/genia/genia.test.iob2
+
+commands:
+  - name: "install"
+    help: "Install dependencies"
+    script:
+      - "pip3 install --user -r requirements.txt"
+      - "python3 -m spacy download en_core_web_lg"
```

Comment:
I'd use `pip` and `pyton` throughout this file - I think by now everyone's expected to be using 3.x, and it's more universal